loops.js

```diff
@@ -17,8 +18,10 @@
 // loop...
 // klaar!
  // ==========================================
-
-
+for ( i = 0; i < 4; i++ ) {
+    console.log("loop...");
+}
+    console.log("klaar!");
```

Comment:
In de uitwerking werken ze het met if/else, maar deze vind ik compacter en werkt ook gewoon, prima!